scripts/check_macos.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+# Checks that  README.md and the macos github action are up to date with macos_instructions.yml
+
+# This script must be run in the root of the directory where you cloned the dockstore/dockstore repo
+
+CURRENT_FILE_NAME_README=README.md
+TEMP_FILE_NAME_README=README-TEMP.md
+CURRENT_FILE_NAME_ACTION=.github/workflows/macos_installation_instructions.yml
+TEMP_FILE_NAME_ACTION=macos_installation_instructions-TEMP.yml
+
+
+cp "$CURRENT_FILE_NAME_README" "$TEMP_FILE_NAME_README"
+mv "$CURRENT_FILE_NAME_ACTION" "$TEMP_FILE_NAME_ACTION"
```

Comment:
Copying is correct, as `./scripts/macos-instructions.sh` directly modifies `README.md`.

Reply:
I also added this comment explaining why I copied as opposed to moved `README.md`.

```
# Copy "$CURRENT_FILE_NAME_README" as ./scripts/macos-instructions.sh modifies "$CURRENT_FILE_NAME_README"
```

Reply:
Oops, I meant on line 21 `mv "$CURRENT_FILE_NAME_ACTION" "$TEMP_FILE_NAME_ACTION"` is renaming instead of copying. I think it should be `cp "$CURRENT_FILE_NAME_ACTION" "$TEMP_FILE_NAME_ACTION"`

Reply:
The action file is over written/created each time `scripts/macos-instructions.sh` is run. 

https://github.com/dockstore/dockstore/blob/6231e16e27d4d6e58a427729b4c086d3bd66ed6d/scripts/macos-instructions.sh#L32